nntrainer/optimizers/adam.h

```diff
@@ -57,8 +57,14 @@ class Adam : public OptimizerImpl {
    */
   void setProperty(const std::string &key, const std::string &value);
 
+  /**
+   * @copydoc Optimizer::addOptimizerVariable(TensorDim dim)
+   */
+  std::vector<TensorDim> addOptimizerVariable(TensorDim &dim) override;
+
   /**
    * @copydoc Optimizer::addOptimizerVariable(std::vector<Weight> &params)
+   * @todo      Remove this function
    */
   void addOptimizerVariable(std::vector<Weight> &params);
```

Comment:
i think you can remove this function in this PR itself.

Reply:
Removed